client/src/components/Gallery.component.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from "react";
+import { FaArrowLeft, FaArrowRight, FaTimes } from "react-icons/fa";
+import gallery_image1 from "../assets/gallery1.jpg";
+import gallery_image2 from "../assets/gallery2.jpg";
+import gallery_image3 from "../assets/gallery3.jpg";
+import gallery_image4 from "../assets/gallery4.jpg";
+import gallery_image5 from "../assets/gallery5.jpg";
+import gallery_image6 from "../assets/gallery6.jpg";
+
+function Gallery() {
+  const poze = [
+    { image: gallery_image1 },
+    { image: gallery_image2 },
+    { image: gallery_image3 },
+    { image: gallery_image4 },
+    { image: gallery_image5 },
+    { image: gallery_image6 },
+  ];
```

Comment:
From a code perspective, it's nearly perfect. I'd like you to create a vector of images in a slightly different format, one that includes a specific type.

```
interface ImageInfo {
  image: string;
  alt: string;
}

const images: ImageInfo[] = [
  {
    image: "gallery_image1",
    alt: "description"
  },
  // Add more image objects as needed
];
```